Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
```

Comment:
Testing reply.

---
In reply to: [59594953](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r59594953) [](ancestors = 59594953)